Makefile

```diff
@@ -52,7 +52,7 @@ image:
 		{                                                                   \
 		set -e ;                                                            \
 		for i in $(STAGINGVERSION) ;                                        \
-			do docker build --build-arg DRIVERBINARY=$(DRIVERBINARY) -t $(STAGINGIMAGE):$${i} .; \
```

Comment:
maybe add a comment above that for cloud build jobs, GCP_FS_CSI_STAGING_VERSION is going to be set by PULL_BASE_REF plumbed in from above.

Reply:
done